### PR TITLE
Bump runtime and update to 4.2.1

### PR DESCRIPTION
**Pull request by bbhtt**
20.08 will be EOL'd in August, ideally we want 21.08 but the Electron baseapp doesn't have a 21.08 branch right now. I'll update it once the branch is created.

I couldn't find the official release date anywhere or the release notes. https://help.airtame.com/product-roadmap-and-changes/product-changes/product-changes-overview seems to be gone.

cc @klausenbusk, please review when you have time. Thanks!

### CODE REVIEW

**Comment by klausenbusk** [comment]
The changelog can be found here: https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application-, can you update the appdata with the 4.1.1 changelog?

File: com.airtame.Client.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "app-id": "com.airtame.Client",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "19.08",
+  "runtime-version": "20.08",
```
Comment (klausenbusk) [inline com.airtame.Client.json]: I guess this can be bumped to 21.08?
```suggestion
  "runtime-version": "21.08",
```

Comment (bbhtt) [reply com.airtame.Client.json]: Yep thanks missed that.

**Comment by bbhtt** [comment]
> The changelog can be found here: https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application-, can you update the appdata with the 4.1.1 changelog?

Will do. Changelog indicates there is a newer version 4.2.1 but here I get https://airtame.com/download/ 4.1.1

https://downloads-cdn.airtame.com/app/latest/linux/Airtame-4.2.1.deb is available though I guess they didn't update the download page?

File: com.airtame.Client.appdata.xml [hunk 2]
```diff
@@ -26,6 +26,40 @@
   </categories>
   <!-- https://help.airtame.com/product-roadmap-and-changes/product-changes/product-changes-overview -->
```
Comment (klausenbusk) [inline com.airtame.Client.appdata.xml]: Let's update this link.
```suggestion
  <!-- https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application- -->
```

Comment (bbhtt) [reply com.airtame.Client.appdata.xml]: Ah thanks.

**Comment by bbhtt** [comment]
bot, build

(cleaned up the commits)

File: com.airtame.Client.json [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
-  "base": "io.atom.electron.BaseApp",
-  "base-version": "19.08",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "21.08",
   "command": "airtame",
+  "separate-locales": false,
   "finish-args": [
     "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
```
Comment (klausenbusk) [inline com.airtame.Client.json:13]: Why are these needed now? 🤔

Comment (bbhtt) [reply com.airtame.Client.json:13]: If it uses Electron it almost always needs access to dri for graphics rendering (because Electron itself needs it).

```
info: [core-streamer]: [Streamer] Creating streamer 0x2a11af575d20 [version: v2.0.0]
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri, suffix _dri)
libGL error: failed to load driver: amdgpu
[46:0809/083333.564331:ERROR:buffer_manager.cc(488)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from pr
```

No idea about pulseaudio but in the logs I see that it is trying to look for some audio devices for capturing probably.

```
info: [core-streamer]: [audio_capture_rtaudio] Init called
info: [core-streamer]: [audio_capture_rtaudio] Found 1 audio devices: 
info: [core-streamer]: [audio_capture_rtaudio] device 0, name: PulseAudio, status: available, out_channels: 2, in_channels: 2
```

**Comment by klausenbusk** [comment]
Thanks for the PR :)
